cmd/terramate/cli/telemetry/endpoint_localhost.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 Terramate GmbH
+// SPDX-License-Identifier: MPL-2.0
+
+//go:build localhostEndpoints
```

Comment:
I'm using the same approach as the checkpoint API here. It could be more convenient for debugging, but beyond that there is no reason to make the endpoint address configurable, so I'd just go with the existing scheme.